src/features/pastelPaperWalletGenerator/PastelPaperWalletModal.tsx

```diff
@@ -14,7 +14,17 @@ import styles from './PastelPaperWalletModal.module.css'
 import { closePastelPaperWalletModal } from './PastelPaperWalletModalSlice'
 import QRCodeGEnerator from './QRCodeGEnerator'
 
-export default function PastelPaperWalletModal(): JSX.Element {
+type InfoProps = {
+  currencyName: string
+}
+
+type PastelPaperWalletModalProps = {
+  info: InfoProps
+}
```

Comment:
Since `PastelPaperWalletModalProps` is only used in this component, it should be just `Props`, according to our convention of naming types of props for react components
```suggestion
type Props = {
  info: {
    currencyName: string
  }
}
```

Reply:
It has been updated. Please check  @br4in3x